src/auth-service/config/global/email-templates.js

```diff
@@ -7,7 +7,7 @@ const logger = log4js.getLogger(
 );
 
 const emailTemplates = {
-  EMAIL_GREETINGS: function (name) {
+  EMAIL_GREETINGS: function ({ name }) {
```

Comment:
Thanks @BenjaminSsempala :
Your `EMAIL_GREETINGS` function is nicely structured. To further improve its reliability, especially in handling edge cases, suggest changing it to `EMAIL_GREETINGS: function ({ name } = {}) { }`. This modification ensures the function can be called even if no arguments are provided, enhancing its versatility. Excellent job so far!

Reply:
Thanks @Baalmart I've included the modifications.